src/main/java/uk/gov/ch/repository/update/OverseasEntityManagingOfficersRepository.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.ch.repository.update;
+import java.util.List;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+import uk.gov.ch.model.update.OverseasEntityManagingOfficerData;
+
+public interface OverseasEntityManagingOfficersRepository
+        extends PagingAndSortingRepository<OverseasEntityManagingOfficerData, Long> {
+
+    @Query(value = "SELECT cba.officer_id AS MO_ID, " +
+            "od.officer_date_of_birth AS DOB, " +
```

Comment:
I would recommend getting one of our CH DBAs to review the sql - either Mike Williams or Prathima Sathyanarayana, to check the full outer joins won't cause performance issues when used with the live dataset. We have had some queries takes minutes to return in previous projects causing connection timeouts.

Reply:
Talked with Mike Williams and got a new query implemented 😄